ktor-server/ktor-server-plugins/ktor-server-websockets/jvm/src/io/ktor/server/websocket/DurationsJvm.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2014-2024 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.websocket
+
+import java.time.*
+
+/**
+ * Ping interval or `null` to disable pinger. Please note that pongs will be handled despite this setting.
+ */
+@Deprecated("Use variant with kotlin.time.Duration", level = DeprecationLevel.ERROR)
+public inline var DefaultWebSocketServerSession.pingInterval: Duration?
```

Comment:
Now I'm not sure that it worth to keep these extensions with deprecation as they could confuse users 🤔